src/R/Editor/Impl/Settings/REditorSettings.cs

```diff
@@ -22,125 +16,73 @@ public static class REditorSettings {
+        public bool PartialArgumentNameMatch {
+            get { return Storage.Get(PartialArgumentNameMatchKey, false); }
+            set { WritableStorage?.Set(PartialArgumentNameMatchKey, value); }
         }
 
-        public static IndentType IndentType {
-            get { return (IndentType)_storage.GetInteger(CommonSettings.FormatterIndentTypeKey, (int)IndentType.Spaces); }
-            set { WritableStorage?.SetInteger(CommonSettings.FormatterIndentTypeKey, (int)value); }
+        public bool EnableOutlining {
+            get { return Storage.Get(EnableOutliningKey, true); }
+            set { WritableStorage?.Set(EnableOutliningKey, value); }
         }
 
-        public static int IndentSize {
-            get { return _storage.GetInteger(CommonSettings.FormatterIndentSizeKey, 4); }
-            set { WritableStorage?.SetInteger(CommonSettings.FormatterIndentSizeKey, value); }
-        }
-
-        public static IndentStyle IndentStyle {
-            get { return (IndentStyle)_storage.GetInteger(CommonSettings.IndentStyleKey, (int)IndentStyle.Smart); }
-            set { WritableStorage?.SetInteger(CommonSettings.IndentStyleKey, (int)value); }
-        }
-
-        public static int TabSize {
-            get { return _storage.GetInteger(CommonSettings.FormatterTabSizeKey, 4); }
-            set { WritableStorage?.SetInteger(CommonSettings.FormatterTabSizeKey, value); }
-        }
-
-        public static bool SendToReplOnCtrlEnter {
-            get { return _storage.GetBoolean(REditorSettings.SendToReplOnCtrlEnterKey, true); }
-            set { WritableStorage?.SetBoolean(REditorSettings.SendToReplOnCtrlEnterKey, value); }
-        }
-
-        public static bool SyntaxCheckInRepl {
-            get { return _storage.GetBoolean(REditorSettings.SyntaxCheckInReplKey, false); }
-            set { WritableStorage?.SetBoolean(REditorSettings.SyntaxCheckInReplKey, false); }
-        }
-
-        public static bool PartialArgumentNameMatch {
-            get { return _storage.GetBoolean(REditorSettings.PartialArgumentNameMatchKey, false); }
-            set { WritableStorage?.SetBoolean(REditorSettings.PartialArgumentNameMatchKey, value); }
-        }
-
-        public static bool EnableOutlining {
-            get { return _storage.GetBoolean(REditorSettings.EnableOutliningKey, true); }
-            set { WritableStorage?.SetBoolean(REditorSettings.EnableOutliningKey, value); }
-        }
-
-        public static RFormatOptions FormatOptions {
+        public RFormatOptions FormatOptions {
             get {
-                _formatOptions.IndentSize = REditorSettings.IndentSize;
-                _formatOptions.IndentType = REditorSettings.IndentType;
-                _formatOptions.TabSize = REditorSettings.TabSize;
-
+                _formatOptions.IndentSize = IndentSize;
+                _formatOptions.IndentType = IndentType;
+                _formatOptions.TabSize = TabSize;
```

Comment:
Is it intentional that you return one and the same instance of `RFormatOptions`?